templates/display_results.html

```diff
@@ -0,0 +1,29 @@
+{% extends 'base.html' %}
```

Comment:
Is this template used to display search results? If so might make sense to call it `search_results.html`

Reply:
Will do